estate/models/estate_property.py

```diff
@@ -0,0 +1,82 @@
+from odoo import models, fields, api
+from dateutil.relativedelta import relativedelta
+from datetime import date
+from odoo.exceptions import UserError, ValidationError
+
+
+class estate_property(models.Model):
+    _name = "estate_property"
```

Comment:
```suggestion
    _name = "estate.property"
```
we use . while giving model name as per convention make sure to check in other files also.